phpunit.xml.dist

```diff
@@ -2,11 +2,8 @@
 <phpunit bootstrap="./tests/bootstrap.php" colors="true">
 
     <testsuites>
-        <testsuite name="unit">
-            <directory suffix="Test.php">./src/Sulu/Component/*/Tests/Unit</directory>
-        </testsuite>
-        <testsuite name="functional">
-            <directory suffix="Test.php">./src/Sulu/Component/*/Tests/Functional</directory>
+        <testsuite name="components">
```

Comment:
why is that change necessary? i find it useful that first all unit tests run and the the "longer" running functional tests.